src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the class definition.